File: Data Structures & Algorithms/is-anagram/submission-1.go
func isAnagram(s string, t string) bool {
    if len(s) != len(t) {
        return false
    }

    var frequency = [26]int{}

    for i := 0; i < len(s); i++ {
        frequency[s[i]-'a']++
        frequency[t[i]-'a']--
    }

    for _, count := range frequency {
        if count != 0 {
            return false
        }
    }

    return true
}

File: Data Structures & Algorithms/count-number-of-islands/submission-1.py
class Solution:


    def numIslands(self, grid: List[List[str]]) -> int:
        count = 0

        def islandfinder(r, c):

            if (r < 0 or r >= len(grid) or c < 0 or c >= len(grid[0])):
                return

            if grid[r][c] in '0':
                return
    
            grid[r][c] = '0'

            islandfinder(r+1, c)
            islandfinder(r-1, c)
            islandfinder(r, c+1)
            islandfinder(r, c-1)

        for i in range(len(grid)):
            for j in range(len(grid[0])):
                if grid[i][j] == '1':
                    count += 1
                    islandfinder(i, j)

        return count